components/server-side-render/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Component,
+	RawHTML,
+} from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+export class ServerSideRender extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			response: {},
```

Comment:
Is the empty object as a representation of missing state a pattern from somewhere else? I'd expect `null` or `undefined` to be a better fit. This works well with the fact that the request resolution already checks for response truthiness:

```js
if ( response && response.output ) {
```

Meaning that we can easily distinguish the three possible states, e.g.:
- no content: `response === null`
- content missing: `response !== null && isEmpty( response )`
- count found: `! isEmpty( response )`